cli/cmd/cli_unix.go

```diff
@@ -41,7 +48,17 @@ func (c *cliState) UpdateCommand() string {
   brew upgrade lacework-cli
 `
 	}
+
+	if isCloudshell() {
+		return `
+  curl https://raw.githubusercontent.com/lacework/go-sdk/main/cli/install.sh | bash -s -- -d $HOME/bin
+`
+	}
 	return `
   curl https://raw.githubusercontent.com/lacework/go-sdk/main/cli/install.sh | bash
 `
 }
+
+func isCloudshell() bool {
+	return os.Getenv(gcpCloudEnv) == "true" || os.Getenv(awsCloudEnv) == "Cloudshell" || os.Getenv(AzureCloudEnv) == "CloudShell"
```

Comment:
Could we split these checks into three different functions?
* `isAwsCloudShell()`
* `isGcpCloudShell()`
* `isAzureCloudShell()`

These functions has to explain what it does (check if running on XYZ cloud shell) and what to we expect.

Then this function should call them all:
```suggestion
	return isAwsCloudShell() || isGcpCloudShell() || isAzureCloudShell()
```